smac/facade/smac_facade.py

```diff
@@ -43,9 +47,10 @@ def __init__(self,
                  runhistory: RunHistory=None,
                  intensifier: Intensifier=None,
                  acquisition_function: AbstractAcquisitionFunction=None,
-                 model:AbstractEPM=None,
+                 model: AbstractEPM=None,
                  runhistory2epm: AbstractRunHistory2EPM=None,
                  initial_design: InitialDesign=None,
+                 initial_configurations: typing.List[Configuration]=None,
```

Comment:
This was also my initial feeling, but the initial designs need a lot of objects which will be created in the SMAC facade. If I would create the initial design object in the smac_cli, I would need to duplicate a lot of code from the SMAC facade. Since I would like to avoid this, I would prefer to keep it this way.